dts/arm/st/f7/stm32f7.dtsi

```diff
@@ -633,6 +633,24 @@
 			has-vref-channel;
 		};
 
+		adc2: adc@40012100 {
+			compatible = "st,stm32-adc";
+			reg = <0x40012100 0x400>;
```

Comment:
Is this memory range correct? Shouldn't it be something like `<0x40012100 0x50>, <0x40012300 0x0c>` (similar for adc1-adc3), or is that too fine-grained? I don't know what the length field is actually used for in zephyr. From the RM:

### Table 101. ADC global register map

Offset | Register
-|-
0x000 - 0x04C | ADC1
0x050 - 0x0FC | Reserved
0x100 - 0x14C | ADC2
0x118 - 0x1FC | Reserved
0x200 - 0x24C | ADC3
0x250 - 0x2FC | Reserved
0x300 - 0x308 | Common registers

Reply:
See discussion here, length of `0x50` seems to be correct: https://github.com/zephyrproject-rtos/zephyr/pull/31771#discussion_r566788513